src/PrivateDns/PrivateDns/help/Get-AzPrivateDnsRecordSet.md

```diff
@@ -142,8 +142,8 @@ This command gets an array of all record sets in the private zone named myzone.c
 
 ### Example 4: Get all record sets in a private zone, using a PSPrivateDnsZone object
 ```powershell
- $Zone = Get-AzPrivateDnsZone -Name "myzone.com" -ResourceGroupName "MyResourceGroup"
- $RecordSets = Get-AzPrivateDnsRecordSet -Zone $Zone
+$Zone = Get-AzPrivateDnsZone -Name "myzone.com" -ResourceGroupName "MyResourceGroup"
+$RecordSets = Get-AzPrivateDnsRecordSet -Zone $Zone
```

Comment:
If an example has output, it should not assign the result to a variable, because by assigning the value there will be no output.
```suggestion
Get-AzPrivateDnsRecordSet -Zone $Zone
```

Reply:
Can we detect this issue automatically?

Reply:
Our current code can not detect this problem.